packages/temporal-time-utils/README.md

```diff
@@ -145,3 +149,31 @@ await handle.signal(stateSignal, "RUNNING" as ScheduleWorkflowState); // resume
 await handle.cancel(); // stop schedule workflow completely
 await handle.query(stateQuery); // get wf state (running, paused, or stopped)
 ```
+
+## `Entity`
+
+This special class packages an indefinitely long lived Workflow and the Signal and Query that go with updating it. It correctly handles the pending Signals and `continueAsNew`, and calls `continueAsNew` at least once a day as recommended by Temporal.
+
+```ts
+import { Entity } from "temporal-time-utils";
+
+interface Input { /* define your workflow input type here */ }
+interface Update { /* define your workflow update type here */ }
+const entity = new Entity<Input, Update>({
+  activity: 'MyActivityName'
+  activityOptions: {
+    startToCloseTimeout: '1 minute',
+  }
+})
+const handle = await client.start(entity.workflow, {
+  args: [{
+    inputValue: 'initial'
+  }],
+  taskQueue: "scheduler",
+  workflowId: "schedule-for-" + userId,
+});
+
+// during signaling updates
+await client.Signal(entity.Signal, { increment: 1 })
+console.log(await client.Query(entity.Query))
```

Comment:
if we want to be fancy we can even collapse this to `entity.signal()` and `entity.query()` by absorbing the client calls into Entity, but not sure if there would be significant impact on workflow bundle size.